test/bigml/test/api/utils.clj

```diff
@@ -0,0 +1,56 @@
+;; Copyright 2016 BigML
+;; Licensed under the Apache License, Version 2.0
+;; http://www.apache.org/licenses/LICENSE-2.0
+
+(ns bigml.test.api.utils
+  "Contains a few facilities especially meant tor testing"
+  (:require (bigml.api [core :as api])))
+
+(defn authenticated-connection
+  "Returns a properly authenticated connection.
+  Credentials shall be provided through the BIGML_USERNAME
+  and BIGML_API_KEY environment variables"
+  []
+  (let [username (System/getenv "BIGML_USERNAME")
+        api-key (System/getenv "BIGML_API_KEY")]
+    (api/make-connection username api-key)))
+
+(defn- do-verb
```

Comment:
I'd stay away from accessing namespaces this way.  If you want a dynamic way to look up fns like this, I'd suggest a using a map (and keywords are more idiomatic than strings as the keys).  Something like:

```clojure
(ns bigml.test.api.utils
  "Contains a few facilities especially meant tor testing"
  (:require (bigml.api [core :as api]
                       [source :as source]
                       [dataset :as dataset]
                       [model :as model]
                       [prediction :as prediction])))

(def api-fns
  {:source {:create source/create}
   :dataset {:create dataset/create}
   :model {:create model/create}
   :prediction {:create prediction/create}})

(defn- do-verb [verb res-type res-uuid & params]
  (apply (get-in api-fns [res-type verb]) res-uuid params))
```